frontends/web/src/contexts/amount-localization-context.ts

```diff
@@ -0,0 +1,9 @@
+import { createContext } from 'react';
+
+type AmountLocalizationContextProps = {
+    decimal: string;
+    group: string;
+  }
```

Comment:
some day we should check again if eslint can do type spacing maybe this https://eslint.style/rules/ts/type-annotation-spacing